modules/ossm-cr-status.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-reference-smcp.adoc
+
+[id="ossm-cr-status_{context}"]
+= Service Mesh status
+
+The information in your `ServiceMeshControlPlane` file under `status` describes the current state of your service mesh.  This information is generated by the Operator and is not editable.
+
+.`ServiceMeshControlPlane` resource `ControlPlaneStatus`
+|===
+|Name |Description |Type
```

Comment:
jfyi, nothing in status is configurable by the user.  it will always be read-only.